changelog/changed-function-die-struct.md

```diff
@@ -0,0 +1 @@
+The `FunctionDie` struct now stores the actual address `Range`'s for which the DIE is valid. This is important because these are not necessarily contiquous, and checking for inclusion between the former `low_pc` and `high_pc` could give false positives.
```

Comment:
```suggestion
The `FunctionDie` struct now stores the actual address `Range`'s for which the DIE is valid. This is important because these are not necessarily contiguous, and checking for inclusion between the former `low_pc` and `high_pc` could give false positives.
```

Reply:
Also the struct isn't public, why do we need the changelog entries?

Reply:
That's a good question. It was a knee-jerk reaction to add it, but I think it just creates noise, so I'll remove it.